.drone.yml

```diff
@@ -1,13 +1,13 @@
 image: clever/drone-go:1.5
-script:
-  - make test
 notify:
   email:
     recipients:
-      - drone@clever.com
-  hipchat:
-    room: Clever-Dev-CI
-    token: {{hipchatToken}}
-    on_started: true
-    on_success: true
+    - drone@clever.com
+  slack:
     on_failure: true
+    on_started: false
+    on_success: false
+    webhook_url: $$slack_webhook
```

Comment:
Should we add a github publish hook (https://github.com/Clever/clever-cli/blob/master/.drone.yml#L14-L24)